### PR TITLE
Reorganize react dir structure

### PR DESCRIPTION
**Pull request by zbraniecki**

### CODE REVIEW

File: www/src/app/components/report/Report.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import UserReport from './UserReport';
-import { reportName } from '../utils';
+import { reportName } from '../../utils/utils';
```
Comment (stasm) [inline www/src/app/components/report/Report.js]: Please rename this as `utils/index.js` and `import … from ../../utils;` here.

File: www/src/app/app.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```
Comment (stasm) [inline www/src/app/app.js]: Should this be in `www/src/views`?

**Comment by zbraniecki** [comment]
updated the PR to your review

**Comment by stasm** [comment]
thanks so much -- I'm glad that with this we're adhering to the current best practices for react projects!

**Pull request merged**